Jenkinsfile

```
@@ -148,19 +148,23 @@ def fault_test_tag() {
 }
 
 def release_candidate() {
-    if env.ON_TAG {
-        if env.RC_IN_TAG {
-            return true
-        }
+    if (sh(label: "Determine if building (a PR of) an RC",
+          script: "git diff-index --name-only HEAD^ | grep -q TAG && " +
+                  "grep -i '[0-9]rc[0-9] TAG'",
```

Apologies if this one was my error:

```suggestion
                  "grep -i '[0-9]rc[0-9]' TAG",
```